### DISCUSSION TITLE
adguardhome not filters android peripherals

### DISCUSSION THREAD

**Discussion by hpi06** [AdguardTeam/AdGuardHome#5174] [General]
hello.
I installed Adguardhome on a raspberry pi and placed it on my home network.
i entered the IP of the raspberry on my Internet box router, and it works fine for all my computers connected by cable or wifi.
But it's not working on my androids peripherals. 3 smartphones and 1 tablet.
I don't know why only androids peripherals are not filtred like the computers.

Do you have a solution to resolve this problem?

Thank you very much

**Comment by M0UL** [comment]
try manual changing the dns setting to  adguard home ip address on those android devices. i had to do the same thing for android tv device.

**Comment by hpi06** [comment]
I tried. But no filter.
The only solution i found is to install dns changer app on android to force 
the dns.
